devtempstats_service/native/src/impl/DevTempStatsService.hpp

```diff
@@ -0,0 +1,65 @@
+#ifndef DEVTEMPSTATS_SERVICE
+#define DEVTEMPSTATS_SERVICE
+
+#include <aidl/devtempstats/service/api/BnDevTempStatsService.h>
+
+namespace dtssapinamespace = ::aidl::devtempstats::service::api;
+
+namespace devtempstats
+{
+   namespace service
+   {
+
+      class DevTempStatsService
+          : public ::aidl::devtempstats::service::api::BnDevTempStatsService
+      {
+      public:
+         DevTempStatsService();
+         ~DevTempStatsService();
```

Comment:
Daca destructorul nu are implementare putem sa il definim default 
```suggestion
         ~DevTempStatsService() = default
```